src/plugin.cc

```diff
@@ -247,7 +247,7 @@ void plugin::Zeek_Spicy::Plugin::registerFileAnalyzer(const std::string& name,
     // this point already, so ours won't get initialized anymore.
     c->Initialize();
 
-    trackComponent(c, c->Tag().Type()); // must come after Initialize(); needed for Zeek < 4.2
```

Comment:
Not sure what this was about, but the comment seems to say something's not needed anymore now?

Reply:
These `trackComponent` calls update `Plugin::_analyzer_name_to_tag_type` which we seem to be using to update linker scopes for pre-registered analyzers, see e.g., https://github.com/zeek/spicy-plugin/blob/09184f9f707b275c90da50a07ca7466dfc7ecd4c/src/plugin.cc#L163-L178. Not sure, but maybe we'll be able to remove this once we remove plugin JIT mode.